estate/models/estate_property.py

```diff
@@ -48,6 +52,11 @@ class EstateProperty(models.Model):
     total_area = fields.Integer(compute="_compute_total_area")
     best_price = fields.Float(compute="_compute_best_price")
 
+    _sql_constraints = [
+        ("check_expected_price", "CHECK(expected_price > 0)", "The expected price must be strictly positive."),
+        ("check_selling_price", "CHECK(selling_price > 0)", "The selling price must be strictly positive."),
```

Comment:
If an accepted offer is refused, you should set the selling price as 0
```suggestion
        ("check_selling_price", "CHECK(selling_price >= 0)", "The selling price must be strictly positive."),
```

Reply:
Ok. At this point, an offer can be refused after have being accepted. However, in the future, once is accepted or refused, the offer can not change its status. I agree with the proposed change, though.